stratego/lib/stratego/game/game.ex

```diff
@@ -0,0 +1,97 @@
+defmodule Stratego.Game do
+
```

Comment:
There's quite a lot of duplication here where you'll have two functions, one for red and one for blue.

If you kept all the state in a map, indexed by the player, then you could have just one function which handled both.